packages/@o3r/rules-engine/src/engine/ruleset-executor.ts

```diff
@@ -195,38 +195,58 @@ export class RulesetExecutor {
     throw new Error(`Unknown condition block met : ${JSON.stringify(nestedCondition)}`);
   }
 
+  // TODO: make private and move to top of the class in a separate PR to make the diff more meaningful in this one
```

Comment:
Or just do it in a separated commit if you are scared about the diff.
I am not sure this change should request: a "TODO" in the code + a Github issue + a new PR

Reply:
I don't intend to create an issue for that. I still find it cleaner to do it in a separate pull request because
1. the change is not linked to this one
2. if I do it in a separate commit, a reviewer will have to be aware that a certain commit has to be selected in order to review the PR

Reply:
https://github.com/AmadeusITGroup/otter/issues/1044